src/tests/ftest/util/test_utils.py

```diff
@@ -336,24 +336,26 @@ def check_pool_daos_space(self, s_total=None, s_free=None):
             for index, item in enumerate(val)]
         return self._check_info(checks)
 
-    def check_rebuild_status(self, rs_version=None, rs_pad_32=None,
-                             rs_errno=None, rs_done=None,
-                             rs_toberb_obj_nr=None, rs_obj_nr=None,
-                             rs_rec_nr=None):
+    def check_rebuild_status(self, rs_version=None, rs_seconds=None,
```

Comment:
(pylint-too-many-arguments) Too many arguments (11/10)

Reply:
This error can be ignored by adding the following line between lines 352 and 353:
```
        # pylint: disable=too-many-arguments
```